templates/highly-efficient-record-processor.cfg.xml.erb

```diff
@@ -7,7 +7,6 @@
     <template crush="<%= @template_crush %>">
 <%- if @template.nil? -%>
     <![CDATA[
-        <?xml version="1.0"?>
```

Comment:
Yea this goes back to how this block is actually used by the herp filter, it's actually wrapped in an ATOM payload elsewhere so this is just a block of xml